napari/layers/image/_image_constants.py

```diff
@@ -53,10 +53,14 @@ class Rendering(StringEnum):
             * iso: isosurface. Cast a ray until a certain threshold is
               encountered. At that location, lighning calculations are
               performed to give the visual appearance of a surface.
+            * avg: average intensity projection. Cast a ray and display the
+              average of values that were encountered.
     """
 
     TRANSLUCENT = auto()
     ADDITIVE = auto()
     ISO = auto()
     MIP = auto()
     ATTENUATED_MIP = auto()
+    VAR = auto()
```

Comment:
Looks like `VAR` isn't implemented yet

Reply:
oops - nope not yet, the calculation works for it but the contrast limits are completely out of the right range so I need to keep playing. Regardless - should be on a separate branch!